README.md

```diff
@@ -1,75 +1,85 @@
+- [Migration for Integrators](https://www.nimiq.com/developers/migration/migration-integrators): A guide for those who want a more in depth overview of the transitioning process from Proof-of-Work to Proof-of-Stake.
+- [Blockchain Explorer](https://www.nimiqhub.com/): NimiqHub allows you to switch from Testnet to Mainnet.
 
-## System requirements
+## Hardware Requirements per Node Type
 
-- 64-bit computing architecture.
-- 2 or more CPU cores
-- 2GB of available RAM memory
-- 5GB of free disk space, an SSD is highly recommended (100GB or more is required for history nodes)
-- File systems with sparse file support.
-- It is highly recommended to run a clock synchronization protocol such as NTP. This
-  is needed to properly accept blocks according to the timestamp and it is especially
-  important for validators in order to produce blocks in the expected timestamps.
+| PoS Node Type | Memory | CPU | Storage | Network | Syncing Time |
+| --- | --- | --- | --- | --- | --- |
+| **History** | Minimum 16GB RAM (higher recommended) | Minimum 4 vCPUs, 8 recommended | Starts from a few gigabytes, grows linearly with blockchain size | High-speed, reliable internet connection; Good I/O performance (SSDs required) | Sync time increases over the life of the blockchain |
+| **Full** | Minimum 16GB RAM | Minimum 4 vCPUs, 8 recommended | Minimum 80GB of storage, 160GB recommended | High-speed, reliable internet connection; Good I/O performance (SSDs recommended) | Sync time grows linearly but slowly |
+| **Light** | 1GB RAM recommended | 64-bit recommended | Works with minimal storage | Moderate-speed internet connection (1 Mbps or higher) | Syncs in a few seconds |
 
-## Installation
+#### Additional Recommendations:
+- File System: Ensure support for sparse files.
+- Clock Synchronization: Use a protocol like NTP for accurate block acceptance, which is essential for validators to produce blocks on time.
 
-Besides [Rust stable](https://www.rust-lang.org/learn/get-started#installing-rust) itself,
-the following packages are required to be able to compile the source code:
+### Service Nodes Additional Requirements
+Nimiq has also two specific node types with specialized roles in maintaining the network security and performing more advanced tasks.
 
-- `clang`
-- `cmake`
-- `libssl-dev` (in Debian/Ubuntu) or `openssl-devel` (in Fedora/Red Hat)
-- `pkg-config`
+- **Validators** for block production:
+    - **PoS Node Type**: Full or History
+    - **Memory**: 16GB RAM minimum
+    - **CPU**: 4 vCPUs recommended
 
-We currently do not make any guarantees about the minimum supported Rust version to consumers, but we currently test two versions older than the current Rust stable.
+- **Prover nodes** for zero-knowledge proof generataion:
+    - **PoS Node Type**: Full or History
+    - **Memory**: 64GB RAM minimum
+    - **CPU**: 8vCPUs recommended
```

Comment:
This process is CPU bound, so I'd increment the recommended vCPUs to 32 at least.